### ISSUE TITLE
execution steps 20180804

### ISSUE THREAD

**Issue by martomato** [Durkastan/durkabot#22]
okay so here's the deal
- [x] #13 
DATA PERSISTENCE!!!!111!!!1!!!!1!
after that's done, we can move on to DATA PERSISTENCE!!!!1!!1(edited)
then, DATA PERSISTENCE!!!!!1!1!1!1!111!1
did I mention DATA PERSISTENCE!!!!!1!11!!!!!!11!1!?
what this means is data persistence is the no.1 priority
like
:b:ery im:b:ortant
now then..
once data persistence is done, and only then, you can do continuous deployment

- [ ] #17 
CONTINUOUS DEPLOYMENT STARTS HERE(edited)
this thing is a guide from containerizing the thing, to enabling automatic builds, to hosting captainhook and making a docker webhook point to captain hook https://blog.gopheracademy.com/advent-2014/easy-deployment/

basically it's errythang you need... but it's geared towards hosting that blog, so it has some irrelevant stuff, methinks
or you can go commando and do it yourself, with the links posted above

1) link docker with github and enable automatic builds https://docs.docker.com/docker-hub/builds/#link-to-a-hosted-repository-service
2) run and configure this container alongside the bot https://hub.docker.com/r/nilbus/docker-hook/
2a) set a webhook url for said container and keep it handy
2b) set the command to redeploy bot
3) make a new webhook with the above webhook url https://docs.docker.com/docker-hub/webhooks/

personally I'd follow the docs :d

**Comment by martomato** [comment]
@ChemicalInk so the only issues that are dealt with in this execution/step list are #13 and #17  ?

**Comment by martomato** [comment]
(answer: yes)
working on it

**Comment by martomato** [comment]
data persistence complete
